### ISSUE TITLE
Isidora to get github pro setup

### ISSUE THREAD

**Issue by igocmanac** [neurogenomics/project_management#7]
Visit [GitHub's website](https://github.com/), and fill in your details. 

Have a look at [GitHub's documentation](https://docs.github.com/en/get-started/signing-up-for-github/signing-up-for-a-new-github-account) if you need help.

**Comment by igocmanac** [comment]
Am I supposed to pay for this pro upgrade?


![Image](https://github.com/neurogenomics/project_management/assets/149688828/cd2aa11e-3c8a-4a96-9517-e59a7ad3db13)

**Comment by NathanSkene** [comment]
Nope, you should be able to get it as a student for free. Thought above link had required info, if if not look for student free GitHub on google.

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by igocmanac** [comment]
sent a request on Friday, waiting for approval (up to 9 days)
